scripts/setup-prod.ts

```
@@ -259,6 +314,79 @@ function firstExistingEnv(key: string): string | null {
+	if (!m) return false;
+	const [a, b] = [Number(m[1]), Number(m[2])];
+	return a === 10 || (a === 192 && b === 168) || (a === 172 && b >= 16 && b <= 31);
+}
+/**
+ * Set ORIGIN to this device's LAN URL — but only when it's safe to (re)write: a blank /
+ * localhost default, or an existing http LAN-IP origin (so a moved box refreshes to the
+ * new IP). A real https domain the operator set for a TLS-proxy deploy is left untouched.
+ */
+function setLanOrigin(file: string, origin: string): 'set' | 'kept' {
+	const current = envValue(file, 'ORIGIN');
+	let host = '';
```

ESLint

[error] The value assigned to 'host' is not used in subsequent statements.

(no-useless-assignment)